keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -63,64 +62,79 @@ This all prolongs the release process, it is a hard dependency on a small group
+_Note_: The following user stories are using the keywords of the [gherkin language][gherkin].
+
+[gherkin]: https://cucumber.io/docs/gherkin/reference/
+
+#### User Roles
 
-[gherkin]: https://docs.cucumber.io/gherkin/reference/
+[Release Managers] are SIG Release Team Members that are
 
+> ... Kubernetes contributors responsible for maintaining release branches and creating releases by using the tools SIG Release provides.
+
+[release managers]: https://kubernetes.io/releases/release-managers
+
+An [End User] is used in the CNCF sense to refer to consumers of the Kubernetes artefacts built and published by [Release Managers]
+
+[end user]: https://www.cncf.io/enduser/
 
 ```
-Scenario: Enduser installs a kubelet from the stable channel
-  Given a user has configured the officially documented package mirror for stable releases for a specific kubernetes minor version ("the minor") on their machine
-   When they use the system's package manager to query the list of kubelet packages available (e.g. apt-cache policy kubelet)
-   Then they see a list of all stable patch versions of the kubelet that stem from the minor and a preference to install the latest patch version of the kubelet
-    But don't see any alpha, beta, rc or nightly releases of the kubelet from this specific kubernetes minor version
-    And they don't see any packages of any other kubernetes minor release
+Scenario: End User lists the available packages from the stable channel
+  Given an End User has configured the officially documented package mirror for stable releases on their machine
+  When they use their system's package manager to query the list of kubelet packages available (e.g. apt-cache policy kubelet)
+  Then they see a list of all stable patch versions of the kubelet that stem from the minor version and a preference to install the latest patch version of the kubelet
+  But they do not see any alpha, beta, release candidate or nightly releases of the kubelet package from this specific kubernetes minor version
+  And they do not see any packages of any other kubernetes minor releases
 ```
 
 ```
 Scenario: Release tools automatically publish new packages
-  Given a release team member ran `./gcbmgr stage master --build-at-head --nomock`
-    And a release team member ran `./gcbmgr release master --buildversion=${{VERSIONID}} --nomock`
-   When a user inspects the officially documented deb and rpm kubernetes repositories for this specific kubernetes minor version
-   Then they see the newly cut alpha releases published in the alpha channel only
+  Given a Release Manager ran `./krel stage --nomock` and `./krel release --nomock` to create a new pre (alpha, beta, rc) release.
+  When an End User inspects the officially documented deb and rpm kubernetes repositories for this specific kubernetes minor version
+  Then they see the newly cut pre releases published in the dev channel only
 ```
```

Comment:
Perhaps this story should also include stable releases, not only prereleases?

Reply:
Agreed, fixed as suggested.